index.rst

```diff
@@ -55,6 +61,56 @@ These tools are intended for use by the full range of geosciencies community:
 beginners and experts; students and teachers; national centers and university
 research labs.
 
+Nonlinear and Non-Gaussian Data Assimilation Capabilities in DART
+-----------------------------------------------------------------
+
+One of the historical drawbacks of ensemble data assimilation techniques is the
+assumption that the quantities being assimilated obey a normal distribution.
+While this is often a safe assumption -- distributions of temperature and
+pressure can be approximated using a normal distribution -- many quantities
+such as precipitation, snow depth and tracer concentration, as well as many
+model parameters aren't normally distributed.
+
+Applying traditional ensemble data assimilation techniques in situations where
+assumptions of gaussianity are invalid can lead to poor forecast skill and
+inconclusive results.
+
+To overcome these problems, DART now implements a novel data assimilation
+technique that no longer requires quantities to be normally distributed. The
+Quantile-Conserving Ensemble Filtering Framework :ref:`(QCEFF) <QCEFF>`
+provides a general method of computing increments for the prior ensemble of an
+observed quantity by allowing the use of arbitrary distributions for the prior
+and the observation error. For a detailed description of the QCEFF, see
+Anderson (2022). [2]_ 
+
+While the QCEFF for computing observation increments can lead to significant improvements in 
+analysis estimates for observed variables, those improvements can be lost when using standard 
+linear regression of observation increments to update other state variables. The QCEFF also 
+implements a capability to do regression in a probit probability integral transformed space. 
+Doing the regression of observation quantile increments in the transformed space guarantees 
+that the posterior ensembles for state variables also retain the advantages of the observation space
```

Comment:
This is a good catch. I think the solution is to drop 'other' rather than replacing it with 'unobserved'.  All state variables, observed or unobserved, are updated by regression of the increments for the current observation.

Reply:
fixed in [57652b2](https://github.com/NCAR/DART/pull/588/commits/57652b21c46fd9485862d10073c09485246fc2b1)